recipes/openimageio/3.x/conanfile.py

```diff
@@ -0,0 +1,303 @@
+from conan.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=2.0.0"
+
+
+class OpenImageIOConan(ConanFile):
+    name = "openimageio"
+    description = (
+        "OpenImageIO is a library for reading and writing images, and a bunch "
+        "of related classes, utilities, and applications. There is a "
+        "particular emphasis on formats and functionality used in "
+        "professional, large-scale animation and visual effects work for film."
+    )
+    topics = ("vfx", "image", "picture")
+    license = "Apache-2.0", "BSD-3-Clause"
+    homepage = "http://www.openimageio.org/"
+    url = "https://github.com/conan-io/conan-center-index"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_dicom": [True, False],
+        "with_ffmpeg": [True, False],
+        "with_freetype": [True, False],
+        "with_giflib": [True, False],
+        "with_hdf5": [True, False],
+        "with_libheif": [True, False],
+        "with_libjpeg": ["libjpeg", "libjpeg-turbo"],
+        "with_libjxl": [True, False],
+        "with_libpng": [True, False],
+        "with_libwebp": [True, False],
+        "with_opencv": [True, False],
+        "with_openjpeg": [True, False],
+        "with_openvdb": [True, False],
+        "with_ptex": [True, False],
+        "with_raw": [True, False],
+        "with_tbb": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_dicom": False,  # Heavy dependency, disabled by default
+        "with_ffmpeg": True,
+        "with_freetype": True,
+        "with_giflib": True,
+        "with_hdf5": True,
+        "with_libheif": True,
+        "with_libjpeg": "libjpeg",
+        "with_libjxl": True,  # TODO: Currently produces link failues
+        "with_libpng": True,
+        "with_libwebp": True,
+        "with_openjpeg": True,
+        "with_openvdb": False,  # FIXME: broken on M1
+        "with_opencv": False,
+        "with_ptex": True,
+        "with_raw": False,  # libraw is available under CDDL-1.0 or LGPL-2.1, for this reason it is disabled by default
+        "with_tbb": False,
+    }
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
+    def requirements(self):
+        # Required libraries
+        self.requires("zlib/[>=1.2.11 <2]")
+        self.requires("libtiff/4.7.0")
+        self.requires("imath/3.1.9", transitive_headers=True)
+        self.requires("openexr/3.2.3")
+        if self.options.with_libjpeg == "libjpeg":
+            self.requires("libjpeg/9e")
+        elif self.options.with_libjpeg == "libjpeg-turbo":
+            self.requires("libjpeg-turbo/3.0.4")
```

Comment:
```suggestion
            self.requires("libjpeg-turbo/[>=3.0.3 <4]")
```
(Or .4 if it's actually necessary!)